### PR TITLE
Hammoud/bulletwrapper_get_force()

### PR DESCRIPTION
**Pull request by hammoudbilal**
This pull request includes minor fixes and modifications to `PinBulletWrapper` 
here are the changes

1. modify the demo to match updated `robot_properties_solo` 
2. modify `PinBulletWrapper.get_force()`  to return the correct direction of the force, before, the order of the bodies in `pyBullet.getContactPoints` would choose the robot as bodyB 
3. modify `PinBulletWrapper.get_force()` to return the forces for all end-effectors, non-active  end-effectors get zero force and zero active status. 

[//]: # "Thanks for your contribution.  To make life of the reviewers easier,"
[//]: # "please give this pull request a meaningful title and provide the"
[//]: # "requested information below."

## Description

[//]: # "Description of what you did.  If it is more complex, consider to add a"
[//]: # "'Summary' section."

## How I Tested

[//]: # "Explain how you tested your changes"

## Do not merge before

[//]: # "Link other open pull request here that need to be merged first."
[//]: # "Remove this section if it does not apply."

- [ ] link to dependent pull request

## I fulfilled the following requirements

[//]: # "Please make sure you followed these steps before requesting a review."
[//]: # "Check the boxes in the list below, when done."

- [ ] All new code is formatted according to our style guide (for C++ run clang-format, for Python, run flake8 and fix all warnings).
- [ ] All new functions/classes are documented and existing documentation is updated according to changes.
- [ ] No commented code from testing/debugging is kept (unless there is a good reason to keep it).

### CODE REVIEW

File: src/bullet_utils/wrapper.py [hunk 1]
```diff
@@ -117,6 +117,10 @@ def __init__(
         self.pinocchio_endeff_ids = [
             pinocchio_robot.model.getFrameId(name) for name in endeff_names
         ]
+        # 
+        self.nb_contacts = len(self.pinocchio_endeff_ids)
+        self.contact_status = np.zeros(self.nb_contacts)
+        self.contact_forces = np.zeros([self.nb_contacts, 6])
```
Comment (jviereck) [inline src/bullet_utils/wrapper.py:123]: Why do you make the `contact_status` and `contact_forces` a field on the class?

File: src/bullet_utils/wrapper.py [hunk 2]
```diff
@@ -125,12 +129,10 @@ def get_force(self):
             (:obj:`list` of :obj:`int`): List of active contact frame ids.
             (:obj:`list` of np.array((6,1))) List of active contact forces.
         """
```
Comment (jviereck) [inline src/bullet_utils/wrapper.py]: Please update the comments about the return type of the method as well.

**Review by jviereck — CHANGES_REQUESTED** [review]
Thanks for your changes.

I am not sure if changing the return value from `get_force()` is a good idea as it means breaking all users. Maybe it would be better to create a new `get_contact_forces()` which does what you have implemented?

**Comment by hammoudbilal** [comment]
Alright, I restored the previous `get_force()` method and added a new method called `end_effector_forces()`  and fixed the remaining issues

**Pull request merged**